judgments/tests/test_utils.py

```diff
@@ -95,6 +96,21 @@ def test_press_summary_list_breadcrumbs(self):
             },
         ]
 
+    def test_is_exact_ncn_match_empty_ncn(self):
+        document = JudgmentFactory.build(neutral_citation=None)
+
+        assert not is_exact_ncn_match(document, "foo")
```

Comment:
Really picky, but could this be:
```suggestion
        assert is_exact_ncn_match(document, "foo") is False
```

and similar for the others? That way if it ever suddenly change the signature and it starts returning things like `[]` tests will explode.